src/autobot.ts

```diff
@@ -84,22 +64,44 @@ export interface Hooks {
   };
 }
 
+interface UninitializedPlugin {
+  initialized: false;
+  name: string;
+  plugin: UninstantiatedPlugin;
+}
+interface InitializedPlugin {
+  initialized: true;
+  name: string;
+  plugin: Plugin;
+}
+
+interface PluginCollection {
+  [pluginName: string]: UninitializedPlugin | InitializedPlugin;
+}
```

Comment:
Because PR plugins are instantiated in every PR I had issues where statuses were being send multiple times in dev because it's in server, not a lambda. I've essentially made sure that autobot will only ever have 1 instance of a Plugin at any given time.